packages/account/src/account.ts

```diff
@@ -497,15 +498,17 @@ export class Account extends AbstractAccount {
    */
   async sendTransaction(
     transactionRequestLike: TransactionRequestLike,
-    options?: Pick<ProviderSendTxParams, 'awaitExecution'>
+    options?: ProviderSendTxParams
   ): Promise<TransactionResponse> {
     if (this._connector) {
       return this.provider.getTransactionResponse(
         await this._connector.sendTransaction(this.address.toString(), transactionRequestLike)
       );
     }
     const transactionRequest = transactionRequestify(transactionRequestLike);
-    await this.provider.estimateTxDependencies(transactionRequest);
+    if (options?.estimateTxDependencies === undefined || options.estimateTxDependencies === true) {
```

Comment:
This leaves the default to be to estimate dependencies as it was previously, but it allowed me to pass in `{estimateTxDependencies: false}` for contract calls and simulations and skip the estimation as it was already done beforehand.

Reply:
We can improve this `if` statement following this same approach [here](https://github.com/FuelLabs/fuels-ts/pull/1767/files#r1494974499)